ios/Runner/AppDelegate.swift

```diff
@@ -7,6 +7,7 @@ import Flutter
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?
   ) -> Bool {
+    UIApplication.shared.isStatusBarHidden = false;
```

Comment:
Acho que isto é o comportamento por default.

Reply:
Since I followed some guides, all of them told me to add this line.
I'll test with and without and update this PR